app/models/course.rb

```diff
@@ -1,6 +1,6 @@
 class Course < ApplicationRecord
   COURSE_PARAMS = %i(name description status estimate_time).freeze
-  enum status: {unactive: 0, active: 1, on_progress: 2}
+  enum status: %i(unactive active on_progress)
```

Comment:
sao k để kiểu hash e
để kiểu hash sẽ động hơn nhé, sau thêm 1 status khác thì các enum khác k bị ảnh hg

Reply:
Em dùng với validate inclusion toàn bị rollback chị ạ. Để thế này nó mới chạy

Reply:
Anh Mạnh giúp em fix rồi nha chị